style/amenity-points.mss

```diff
@@ -2307,11 +2308,6 @@
           text-halo-radius: 0;
         }
       }
-      [feature = 'leisure_dog_park'] {
-        text-fill: @leisure-green;
-        text-halo-radius: @standard-halo-radius * 1.5; /* Extra halo needed to stand out from paw pattern. */
-        text-halo-fill: @standard-halo-fill;
-      }
```

Comment:
This removes pattern fill styling and replaces it with the simple fill used by park/village green/recreation_ground